Danki2/Assets/Scripts/Effects/EffectTracker.cs

```diff
@@ -21,6 +21,13 @@ public void ProcessEffects()
         );
     }
 
+    public float ProcessStat(Stat stat, float value)
+    {
+        var processedValue = value;
+        _effects.ForEach(e => e.ProcessStat(stat, processedValue));
```

Comment:
This should be `_effects.ForEach(e => processedValue = e.ProcessStat(stat, processedValue));`

Reply:
Good spot